x-pack/plugins/alerting/public/pages/maintenance_windows/components/maintenance_windows_list.tsx

```diff
@@ -170,20 +167,43 @@ export const MaintenanceWindowsList = React.memo<MaintenanceWindowsListProps>(
           },
         },
       ],
-      [onArchive, onCancel, onCancelAndArchive, onEdit]
+      [isMutatingOrLoading, onArchive, onCancel, onCancelAndArchive, onEdit]
     );
 
     const columns = useMemo(
       () => (readOnly ? COLUMNS : COLUMNS.concat(actions)),
       [actions, readOnly]
     );
 
+    const search: EuiSearchBarProps = useMemo(
+      () => ({
+        filters: [
+          {
+            type: 'custom_component',
+            component: StatusFilter,
+          },
+        ],
+        toolsRight: (
+          <EuiButton
+            data-test-subj="refresh-button"
+            iconType="refresh"
+            onClick={refreshData}
+            isLoading={isMutatingOrLoading}
+            isDisabled={isMutatingOrLoading}
```

Comment:
nit: If they are the same value we probably don't need both. `eui` says that `isLoading` already disables it.

Reply:
TIL!